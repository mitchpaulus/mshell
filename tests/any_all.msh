"Anys" wl
[] (3 <) any str wl
[1 2 3] (3 <) any str wl
[1 2 3] (0 <) any str wl
[false false true] () any str wl
[false false false] () any str wl
[true true true] () any str wl

# Now do alls
"Alls" wl
[] (3 <) all str wl
[1 2 3] (3 <) all str wl
[1 2 3] (0 <) all str wl
[false false true] () all str wl
[false false false] () all str wl
[true true true] () all str wl
