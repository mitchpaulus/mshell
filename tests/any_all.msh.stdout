Anys
false
true
false
true
false
true
Alls
true
false
false
false
false
true
